public-ui/src/app/components/db-no-results/no-results-search.component.tsx

```diff
@@ -0,0 +1,188 @@
+            url += '?search=' + searchValue;
+            domainMatch();
+            navigateByUrl(url);
+        } else if (type === 'survey') {
+            localStorage.setItem('surveyModule', JSON.stringify(domainInfo));
+            url += 'survey/' + surveyIdToRoute[domainInfo.conceptId];
+            url += '?search=' + searchValue;
+            domainMatch();
+            navigateByUrl(url);
+        } else if (type === 'pm') {
+            url += 'physical-measurements/' + '/' + searchValue;
+            navigateByUrl(url);
+        } else if (type === 'fitbit') {
+            url += 'fitbit';
+            localStorage.setItem('searchText', searchValue);
+            navigateByUrl(url);
+        }
+    }
+
+    render() {
+        const {searchValue} = this.props;
+        const {loading, domainInfoResults, surveyModuleResults, pmResults, fitbitResults} = this.state;
+        return (
+            <React.Fragment>
+            <style>{styleCss}</style>
+            <div style={styles.noResults}>
+                { loading ?
+                <div style={styles.loadingDiv}>
+                    <p style={styles.loadingText}>Searching whole site for <strong>{searchValue} </strong></p>
+                    <div style={styles.spinnerDiv}><LoadingDots /></div>
+                </div>
+                : null
+                }
+                {
+                 domainInfoResults.map((domainInfo, index) => {
+                    const key = domainInfo.name + index;
+                    return <div key={key}>{domainInfo.standardConceptCount} results available in the domain:
+                    <a onClick={() => this.handleOnClick(domainInfo, 'ehr')}>{domainInfo.name}</a></div>;
+                 })
+                }
+                {
+                surveyModuleResults.map((surveyInfo, index) => {
+                    const key = surveyInfo.name + index;
+                    return <div key={key}>{surveyInfo.questionCount} related questions in survey:
+                    <a onClick={() => this.handleOnClick(surveyInfo, 'survey')}>{surveyInfo.name}</a></div>;
+                })
+                }
+                {
+                pmResults.map((pmInfo, index) => {
+                   const key = pmInfo.name + index;
+                   return <div key={key}>{pmInfo.standardConceptCount} results available in the domain:
+                   <a onClick={() => this.handleOnClick(pmInfo, 'pm')}>{pmInfo.name}</a></div>;
+                })
+                }
+                {
+                fitbitResults.map((fitbitInfo, index) => {
+                   const key = fitbitInfo.name + index;
+                   return <div key={key}>{fitbitInfo.standardConceptCount} results available in the domain:
+                   <a onClick={() => this.handleOnClick(fitbitInfo, 'fitbit')}>{fitbitInfo.name}</a></div>;
+                })
+                }
+            </div>
+            </React.Fragment>
+        );
+      }
```

Comment:
Nit: indentation throughout render, remove wrapping `()`
```suggestion
        return <React.Fragment>
            <style>{styleCss}</style>
            <div style={styles.noResults}>
                { loading ?
                <div style={styles.loadingDiv}>
                    <p style={styles.loadingText}>Searching whole site for <strong>{searchValue} </strong></p>
                    <div style={styles.spinnerDiv}><LoadingDots /></div>
                </div>
                : null
                }
                {
                     domainInfoResults.map((domainInfo, index) => {
                        const key = domainInfo.name + index;
                        return <div key={key}>
                            {domainInfo.standardConceptCount} results available in the domain:
                            <a onClick={() => this.handleOnClick(domainInfo, 'ehr')}>{domainInfo.name}</a>
                        </div>;
                     })
                }
                {
                    surveyModuleResults.map((surveyInfo, index) => {
                        const key = surveyInfo.name + index;
                        return <div key={key}>
                            {surveyInfo.questionCount} related questions in survey:
                            <a onClick={() => this.handleOnClick(surveyInfo, 'survey')}>{surveyInfo.name}</a>
                        </div>;
                    })
                }
                {
                    pmResults.map((pmInfo, index) => {
                       const key = pmInfo.name + index;
                       return <div key={key}>
                          {pmInfo.standardConceptCount} results available in the domain:
                          <a onClick={() => this.handleOnClick(pmInfo, 'pm')}>{pmInfo.name}</a>
                       </div>;
                    })
                }
                {
                    fitbitResults.map((fitbitInfo, index) => {
                       const key = fitbitInfo.name + index;
                       return <div key={key}>
                          {fitbitInfo.standardConceptCount} results available in the domain:
                          <a onClick={() => this.handleOnClick(fitbitInfo, 'fitbit')}>{fitbitInfo.name}</a>
                       </div>;
                    })
                }
            </div>
        </React.Fragment>;
    }
```

Reply:
Thanks changed this.